Census API to Socrata/PythonCensusApiCityReader.py

```diff
@@ -10,6 +11,7 @@ class PythonCensusApiCityReader(object):
     def __init__(self):
```

Comment:
PEP8 says we should use spaces instead of tabs :) http://legacy.python.org/dev/peps/pep-0008/#tabs-or-spaces